app/src/main/webapp/scripts/users/config.js

```diff
@@ -78,7 +80,8 @@ angular.module('universeadm.users.config', ['ui.router', 'universeadm.navigation
           user: function(userService, $stateParams){
             var username = $stateParams.username;
             return username !== null && username.length > 0 ? userService.get(username) : null;
-          }
+          },
+          $title: function() { return 'Edit User | User Management'; }
```

Comment:
This is even used by the create user view. Create a new state for the `/user/` path with `Create User | User Management` title.